process/smartContract/process.go

```diff
@@ -1779,13 +1784,13 @@ func (sc *scProcessor) createSmartContractResults(
 	lenOutTransfers := len(outAcc.OutputTransfers)
 	if lenOutTransfers == 0 {
 		if callType == vmcommon.AsynchronousCall && bytes.Equal(outAcc.Address, tx.GetSndAddr()) {
-			result := createBaseSCR(outAcc, tx, txHash)
+			result := createBaseSCR(outAcc, tx, txHash, outAcc.Nonce)
 			sc.addVMOutputResultsToSCR(vmOutput, result)
 			return true, []data.TransactionHandler{result}
 		}
 
 		if !sc.flagDeploy.IsSet() {
-			result := createBaseSCR(outAcc, tx, txHash)
+			result := createBaseSCR(outAcc, tx, txHash, outAcc.Nonce)
```

Comment:
Here as well, should be 0 instead of `outAcc.Nonce`.

Reply:
fixed